src/site/topic26.rst

```diff
@@ -119,6 +119,61 @@ Recursively Thinking
         * Return the sorted merged list
 
 
+Complexity Analysis
+-------------------
+
+* A simple way to think about the analysis is to consider
+    * How much work is involved for a single merging of two lists
+    * How many times merging needs to happen
+
+* It can get more nuanced, but this level of detail is sufficient
+
+
+Merging
+^^^^^^^
```

Comment:
This seems a little overexplained. I don't really care if `n >= m`, or the exact worst case. From each comparison that is made, you add 1 element to the overall sorted result list, and those sorted elements are never compared again. Boom, headshot.